application/helpers/lib_helper.php

```diff
@@ -107,15 +84,11 @@ function load_db_team() {
 }
 
 /*
- * This function gets the cached team data.
- * If cookie is deleted, get from the database.
+ * This function gets the cached team data from database.
  */
 if (! function_exists('load_cached_team')) {
     function load_cached_team() {
-        $data = load_cookie_team();
```

Comment:
We should have a check of CI's document. It might provide localStorage access via its context.